package.json

```diff
@@ -54,6 +54,7 @@
     "coverage": "vitest --coverage",
     "upgrade-interactive": "npm-check -u"
   },
+  "dependencies": {},
```

Comment:
added to satisfy the type-checker in `vite.config.ts` as I think it's worth to include `packageJson.dependencies` there from the start to avoid future mistakes